Add news for playtest 20170902

reaperrr opened the pull request.

reaperrr commented:
First draft.

The date is just a placeholder (for now).
I won't have time tomorrow, so if we really wanted to publish the playtest then, someone else would have to finish this.

pchote commented:
Can you please add a paragraph about the Mod SDK?  Suggested wording:

> A new version of the Mod SDK is [also now available](https://github.com/OpenRA/OpenRAModSDK/releases/tag/201708XX) with a few improvements and bug fixes.  The big new feature is the ability to [automatically create installers](https://github.com/OpenRA/OpenRAModSDK/wiki/Generating-Release-Installers) for your mod "in the cloud".  We encourage everyone who is working on mods for the last release to try out the ModSDK and report back any issues you find or suggestions that you may have, before it is too late!

reaperrr commented:
Updated, also added link to download section.

In case this *does* get published today, I replaced the XX with 26 so this could be merged as-is if no other changes are necessary.

pchote commented:
I don't see any chance of this happening today, but tomorrow is hopeful.

pchote commented:
We'll need to include a Battlefield News entry, and @abcdefg30 grumbles about social media when I try to do news posts without screenshots.

pchote commented:
I may not have a lot of time tonight, so I tagged the playtest and did the final Mod SDK changes this afternoon when it was more convenient.

* The frozen changelog link is https://github.com/OpenRA/OpenRA/wiki/Changelog/ed51f80f7b866599a65999aa7d1c0c4204d02da4 (get this the same way you do for PR updates)
* The `GITHUB_PLAYTEST_ID` for `lib/openra.rb` is 7534850 (get this from https://api.github.com/repos/OpenRA/OpenRA/releases).
* The frozen Mod SDK link is https://github.com/OpenRA/OpenRAModSDK/releases/tag/20170827

pchote commented on content/news/playtest-20170826.md:
```diff
@@ -0,0 +1,16 @@
+A month after the previous playtest, we're pleased to announce the new release candidate, playtest-20170826.
+
+In addition to all the new features and improvements the previous playtest brought along, in this new playtest we fixed over 30 issues that were identified through player and modder feedback. Among the more noticable ones were the movement 'stuttering' that ground units were showing, as well as some issues with the new command bar and some other UI hiccups.
```
It would be good to change "the previous playtest" to a link to the last news post.

pchote commented on content/news/playtest-20170826.md:
```diff
@@ -0,0 +1,16 @@
+
+Furthermore, the Red Alert mod in particular received a larger balance overhaul, along with a new map (Agenda). 'Agenda' and the updated 'Great Sahara 2' now serve as examples of more prominent use of civilian 'tech' structures that provide various benefits when captured. Tiberian Dawn's balance received some further polishing as well, making bridges more robust (in particular to light weapons an ...
+
+A new version of the Mod SDK is [also now available](https://github.com/OpenRA/OpenRAModSDK/releases/tag/20170826) with a few improvements and bug fixes.  The big new feature is the ability to [automatically create installers](https://github.com/OpenRA/OpenRAModSDK/wiki/Generating-Release-Installers) for your mod "in the cloud".  We encourage everyone who is working on mods for the last release t ...
```
"before it is too late!" is perhaps a bit to ultimatum-ey.  This could be replaced with "so that we can iron out any remaining issues before the legacy oramod infrastructure is removed.".

pchote commented on content/news/playtest-20170826.md:
```diff
@@ -0,0 +1,16 @@
+
+The new playtest can be [downloaded here](http://www.openra.net/download/).
+
+<div class="about-todo-divider"></div>
```
IMO these aren't really needed.  They were only used in the last post to clearly separate out player vs WIP modder sections.

reaperrr commented:
Updated.

pchote commented on content/gamenews:
```diff
@@ -1,8 +1,8 @@
+	Title: Playtest 20170827 available!
+	Author: Reaperrr
+	Content: We're pleased to announce the new playtest-20170827, the second release candidate for the upcoming OpenRA release. Changes include:\n\n  • Over 30 bugs and other issues fixed\n  • New civilian tech structures and a new map for the RA mod\n  • Improved Red Alert balance\n  • Fine-tuned Tiberian Dawn balance\n  • Updated Mod SDK\n\nHead to www.openra.net for more information!
 2017-08-12 00-00-00:
```
The registration window closes in a couple of days, so I think we might as well remove this now.

pchote commented on content/news/playtest-20170827.md:
```diff
@@ -0,0 +1,19 @@
+<div style="text-align:center" markdown="1">
+![New civilian tech buildings](/images/news/20170827-agenda-civ-tech.png)
+
+The capturable civilian tech structures on the new map Agenda.
```
Would be good if this could mention in a couple of words what the two new tech structures do.

Mailaender commented on content/news/playtest-20170827.md:
```diff
@@ -0,0 +1,19 @@
+</div>
+Tiberian Dawn's balance received some further polishing as well, making bridges more robust (in particular to light weapons and flames) as well as making Orcas and rocket infantry less effective versus infantry in general.
+
+A new version of the Mod SDK is [also now available](https://github.com/OpenRA/OpenRAModSDK/releases/tag/20170827) with a few improvements and bug fixes.  The big new feature is the ability to [automatically create installers](https://github.com/OpenRA/OpenRAModSDK/wiki/Generating-Release-Installers) for your mod "in the cloud".  We encourage everyone who is working on mods for the last release t ...
```
I guess the second `ModSDK` wants to be `Mod SDK` as well?

pchote commented on content/gamenews:
```diff
@@ -1,8 +1,8 @@
+2017-08-27 20-00-00:
+	Title: Playtest 20170827 available!
+	Author: Reaperrr
+	Content: We're pleased to announce the new playtest-20170827, the second release candidate for the upcoming OpenRA release. Changes include:\n\n  • Over 30 bugs and other issues fixed\n  • New civilian tech structures and a new map for the RA mod\n  • Improved Red Alert balance\n  • Fine-tuned Tiberian Dawn balance\n  • Updated Mod SDK\n\nHead to www.openra.net for more information!
```
The second bullet point overflows onto a second line.  Can you change it to just "...and a new map for RA"?

reaperrr commented:
Updated.

reaperrr commented:
As mentioned in https://github.com/OpenRA/OpenRA/pull/13930 it's probably better to tag a new playtest, so this shouldn't be merged for now.

netnazgul commented:
If not a burden, please add a bit of update about RAGL:

To other news, Red Alert Global League public registration is closed now. There is still several days before the new season kicks off, but we can now firmly state that this is definitely the biggest open OpenRA tournament to date with a whooping 70 players registered! Will General FiveAces be ready to confront the now unbeatable Lorrydriver and grab back reign as a triple League champion? Or is it the time for Barf blob to dominate the battlefields? Stay tuned for league updates and match highlights!

I even have a pic to that 
![openra_fiveaces_general](https://user-images.githubusercontent.com/1614714/29957420-09daab7a-8ef6-11e7-98f5-bef47c590aee.jpg)

reaperrr commented:
Updated everything except the changelog link and IDs (there might be 1 or 2 more merges for prep).

reaperrr commented:
Updated changelog link (which most likely won't change anymore), all that's left to do is tag the new playtest and update the ID. I don't know how we get the installer packages, though, otherwise I'd have done so already.

netnazgul commented:
A bit of proof-reading

>>Among the more noticable ones were the movement 'stuttering' that ground units were showing, as well as some issues with the new command bar and some other UI hiccups.

Air units returning to base sporadically was the most annoying.

>>The new capturable civilian tech structures on the new map Agenda.

I'd add "displayed on the new map" to more emphasize that new structures are capturable everywhere, not just on Agenda.

>> and grab back reign as a triple League champion

and grab back the reign as the triple League champion

reaperrr commented:
Applied @netnazgul's suggestions.

reaperrr commented:
Updated. Should be ready to merge, unless I overlooked anything.

reaperrr commented:
(Hopefully) final update.
